front/pages/api/login.ts

```diff
@@ -66,275 +98,281 @@ async function handler(
     return;
   }
 
-  switch (req.method) {
-    case "GET":
-      // `workspaceInvite` is set to a `Workspace` if the query includes a `wId`. It means the user
-      // is going through the flow of whitelisted domain to join the workspace.
-      let workspaceInvite: null | Workspace = null;
-      let isAdminOnboarding = false;
-
-      if (req.query.wId) {
-        workspaceInvite = await Workspace.findOne({
-          where: {
-            sId: req.query.wId as string,
+  if (req.method !== "GET") {
+    return apiError(req, res, {
+      status_code: 405,
+      api_error: {
+        type: "method_not_supported_error",
+        message: "The method passed is not supported, GET is expected.",
+      },
+    });
+  }
+
+  // `workspaceInvite` is set to a `Workspace` if the query includes a `wId`. It means the user
+  // is going through the flow of whitelisted domain to join the workspace.
+  let workspaceInvite: null | Workspace = null;
+  let isAdminOnboarding = false;
+
+  const { inviteToken, wId } = req.query;
+  // TODO(2024-01-16 flav) Remove once auto join whitelisted domains is fully released.
```

Comment:
We will still support invites no?

Reply:
We will still support member invite, but won't support workspace invite anymore.